index.d.ts

```diff
@@ -0,0 +1,60 @@
+export interface Options {
+  /**
+   * Speed 10 has 5% lower quality, but is about 8 times faster than the default. Speed 11 disables dithering and lowers compression level.
+   * 
+   * @default 3
+   */
+  speed?: number;
+
+  /**
+   * Remove optional metadata.
+   * 
+   * @default false
+   */
+  strip?: boolean;
+
+  /**
+   * Instructs pngquant to use the least amount of colors required to meet or exceed the max quality. If conversion results in quality below the min quality the image won't be saved.
+   * Min and max are numbers in range 0 (worst) to 1 (perfect), similar to JPEG.
+   * 
+   * @example [0.3, 0.5]
+   * 
+   */
+  quality?: number[];
```

Comment:
This should be a tuple.
```suggestion
  quality?: [number, number];
```